assets/helpers/abTests/abtestDefinitions.js

```diff
@@ -34,4 +34,21 @@ export const tests: Tests = {
     canRun: () => !isFromEpicOrBanner,
   },
 
+  globalContributionTypes: {
+    variants: [
+      'control',
+      'default-annual',
+      'default-annual_no-monthly',
+    ],
+    audiences: {
+      ALL: {
+        offset: 0,
+        size: 1,
+      },
+    },
+    isActive: true,
+    independent: true,
+    seed: 5,
```

Comment:
to your point about excluding people from the test who have URL overrides: let's check this in the `canRun`